impeller/entity/entity_pass_clip_stack.cc

```diff
@@ -52,128 +51,124 @@ EntityPassClipStack::GetClipCoverageLayers() const {
   return subpass_state_.back().clip_coverage;
 }
 
-EntityPassClipStack::ClipStateResult EntityPassClipStack::ApplyClipState(
-    Contents::ClipCoverage global_clip_coverage,
-    Entity& entity,
-    size_t clip_height_floor,
-    Point global_pass_position) {
+EntityPassClipStack::ClipStateResult EntityPassClipStack::RecordRestore(
```

Comment:
Rather that work generic over entities, there is now a RecordClip and RecordRestore operation.